docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -87,7 +87,7 @@ memory availability. Supports the following values:
 
 The amount of time a query is allowed to recover between running out of memory
 and being killed, if ``query.low-memory-killer.policy`` or
-``task.low-memory-killer.policy`` is set to value differnt than ``none``.
+``task.low-memory-killer.policy`` is set to value different than ``none``.
```

Comment:
Because this is an unrelated change, this should be pulled out into a separate commit.